pyaedt/edb_core/siwave.py

```diff
@@ -703,58 +697,13 @@ def add_siwave_ac_analysis(
         Parameters
         ----------
         accuracy_level : int, optional
-           Level of accuracy. The default is ``1``.
+           Level of accuracy of SI Slider. The default is ``1``.
         decade_count : int
-            The default is ``10``.
-        sweeptype : int, optional
-            Type of the sweep. The default is ``1``.
-        start_freq : float, optional
-            Starting frequency. The default is ``1``.
-        stop_freq : float, optional
-            Stopping frequency. The default is ``1e9``.
-        step_freq : float, optional
-            Frequency size of the step. The default is ``1e6``.
-        discrete_sweep : bool, optional
-            Whether the sweep is discrete. The default is ``False``.
-
-        Returns
-        -------
-        bool
-            ``True`` when successful, ``False`` when failed.
-        """
-        self._siwave_setup.AddACSimSetup(
-            self._cell,
-            accuracy_level,
-            str(decade_count),
-            sweeptype,
-            str(start_freq),
-            str(stop_freq),
-            str(step_freq),
-            discrete_sweep,
-        )
-        return self.create_exec_file(add_ac=True)
-
-    @pyaedt_function_handler()
-    def add_siwave_syz_analysis(
-        self,
-        accuracy_level=1,
-        decade_count=10,
-        sweeptype=1,
-        start_freq=1,
-        stop_freq=1e9,
-        step_freq=1e6,
-        discrete_sweep=False,
-    ):
-        """Add a SIwave SYZ analysis.
-
-        Parameters
-        ----------
-        accuracy_level : int, optional
-           Level of accuracy. The default is ``1``.
-        decade_count : int, optional
-            Number of points to calculate in each decade. The default is ``10``.
+            The default is ``10``. Value used for Linear Count and Decade Count Sweep types.
```

Comment:
```suggestion
            The default is ``10``. The value for this parameter is used for these sweep types:
            linear count and decade count.
```
It looks like you accidentally deleted the description portion.